pkg/modules/api/context.go

```diff
@@ -193,7 +193,10 @@ func (ctx *Context) FormData() *FormData {
 
 // GeneratePath generates a path within the context's working directory. It
 // does not create a file.
-func (ctx *Context) GeneratePath(extension string) string {
+func (ctx *Context) GeneratePath(extension string, name ...string) string {
```

Comment:
Is there a better way to do optional params? This is heavily used throughout the code base and I wanted to specify it just for the screenshotting portion rather than unknowable uuid file names

Reply:
You could accept a pointer to an `Options` struct. But if you don't want to change the signature, then this is the only way I know how. (You might want to add a comment explaining this hack.)